CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.2.1]
```

Comment:
I think since this includes some non-backwards compatible changes to `UserInfoKeys`, this should at least be a minor version bump, right?

Reply:
Those keys are only used at runtime inside the user info dictionary passed around through XCGLogger, there's not like a `UserDefaults` usage so there shouldn't be any version compatibility issues.